src/lab12.py

```diff
@@ -719,6 +721,67 @@ def handle_quit(self):
 
 @wbetools.patch(Chrome)
 class Chrome:
+    def paint(self):
```

Comment:
Why does this code return?

Reply:
Oh, you're using `active_tab_url` instead of `Tab.url`, got it, great, good bug fix.